src/FamilyHubs.Referral.Core/Commands/ProcessGridEventCommand.cs

```diff
@@ -197,45 +197,63 @@ private T Deserialize<T>(string? json)
 
     private async Task ProcessUserAccount(UserAccountDto userAccountDto, CancellationToken cancellationToken)
     {
-        //Process Custom Event
-        _logger.LogInformation("Processing User Account Event Grid event message");
+        try
+        {
+            //Process Custom Event
+            _logger.LogInformation("Processing User Account Event Grid event message");
 
-        _logger.LogInformation($"Creating User Account for Processing Events: {userAccountDto.Name}-{userAccountDto.EmailAddress}");
+            _logger.LogInformation($"Creating User Account for Processing Events: {userAccountDto.Name}-{userAccountDto.EmailAddress}");
```

Comment:
You could use named placeholders in the error message and pass the parameters, rather than creating an error message string.

Reply:
Modified Log Message